README.md

```diff
@@ -39,101 +34,95 @@
+Scramjet Cloud Platform is a Serverless End-to-End Distributed Data Transformation Platform working in the Transform-as-a-Service model that can by itself acquire, transform and process data in real-time pipelines spanning across environments in different physical locations and cloud infrastructure providers.
 
-STH allows you to deploy and run multiple data processing apps called Sequences.
+The platform is used for uploading and executing programs, connecting their outputs with inputs using Topics and finally accessing the outputs via a centrally available API. You don't need to set up servers, operating systems and such, just write your program, package it, send it to us and let us know when you want them to start. We'll take care of the rest.
 
-<details>
-<summary>
-    <strong>Sequences</strong>
-</summary>
+Scramjet Cloud Platform allows you to:
 
-[**Sequences**](dictionary.md#sequence) are specific apps, not just any apps. They specialize in efficient data processing.
+* write and deploy simple long-running data processing programs called Sequences,
+* invoke Sequences with a simple CLI or programmatically via API or a set of client programs,
+* monitor and control running Sequences via one central, publicly available, secure API,
+* send and receive data produces and required by running Sequences,
+* share data between multiple Sequences by enclosing them in Spaces,
+* connect self-hosted servers to existing Data Spaces with minimum configuration,
+* create execution environments at multiple cloud providers with a click of a button.
 
-We named our apps "Sequences" and that term describes well its nature, as they process data through a Sequence of chained functions. Therefore, usually our Sequences are concise, easy to write and powerful at the same time.
-</details>
-<br>
+### **1.2. Platform Quick Start**
 
-The core part of our STH engine is called the "host".
+> **Deploy** and **execute** long running data **programs** that, **​transform** **in real-time** and transport **data** ​between multiple clouds and **on-premise** machines, just as **simply** as starting a program on a laptop, ​with no tedious configurations, VPNs or DevOps work.​
 
-<details>
-<summary>
-    <strong>Host</strong>
-</summary>
+#### Log in to Scramjet Cloud Platform Beta Panel
 
-**Host** is responsible for maintaining and deploying Sequences, keeping them running and managing its lifecycle.
+<div className="w-200"></div>
 
-Host exposes also its own REST API to provide and receive data and manage Sequences and host itself.
+| ![Scramjet Welcome Page](images/welcome-page.png) |  To use our platform **free of charge for a year**, please [sign up for the beta program](https://scr.je/join-beta-docs). <br/> If you are already our user, please log in to the [Scramjet Cloud Platform Beta Panel](https://console.beta.scramjet.cloud/) through your provided mail and set the new password by clicking `Forgot the password`. |
```

Comment:
```suggestion
| ![Scramjet Welcome Page](images/cp-welcome-page.png) |  To use our platform **free of charge for a year**, please [sign up for the beta program](https://scr.je/join-beta-docs). <br/> If you are already our user, please log in to the [Scramjet Cloud Platform Beta Panel](https://console.beta.scramjet.cloud/) through your provided mail and set the new password by clicking `Forgot the password`. |
```
I corrected the `.png` file name, but still, this file does not work, @MichalCz you need to upload it again
![image](https://user-images.githubusercontent.com/61054769/189297137-83d480e1-47fd-4b94-a979-ee393afe2929.png)